tools/serve/serve.py

```diff
@@ -48,6 +48,14 @@ def replace_end(s, old, new):
     return s[:-len(old)] + new
 
 
+def domains_are_distinct(a, b):
+    a = a.split(".")
+    b = b.split(".")
+    l = -1 * min(len(a), len(b))
```

Comment:
1. `flake8` doesn't like `l` as an identifier (given ambiguity with `I` and `1`).

2. Can we call the min length something like `shortest`?

3. And then add a separate variable something like `slice` where it gets multiplied by -1? At the moment I find this code a touch confusing.

Reply:
Agree completely. Also, I typically try not to repurpose existing variables. I guess I just got sloppy. One thing: `slice` is the name of a built in, so this latest version uses `slice_index`.